modules/base/src/test/scala/io/janstenpickle/trace4cats/base/context/IdInstanceSummonTest.scala

```diff
@@ -0,0 +1,9 @@
+package io.janstenpickle.trace4cats.base.context
+
+object IdInstanceSummonTest {
```

Comment:
@mrdziuban Summon tests - we just check that our instance is discoverable by the compiler